hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/expression/TestHoodieSparkExpressionIndex.java

```diff
@@ -127,6 +130,7 @@ public void testApplyYearFunctionWithWrongNumberOfArguments() {
     HoodieSparkExpressionIndex index = new HoodieSparkExpressionIndex(
         "yearIndex",
         "year",
+        PARTITION_NAME_COLUMN_STATS,
```

Comment:
Add a new test on `PARTITION_NAME_BLOOM_FILTERS` too?

Reply:
was already covered in functional test, but now I have added one UT here too.